docs/app/Examples/elements/Rail/Types/RailDividingExample.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Grid, Rail, Segment } from 'stardust'
+
+const RailDividingExample = () => (
+  <Grid className='three column'>
+    <Grid.Column className='centered'>
+      <Segment>
+        <Rail dividing position='left'>
+          <Segment>Left Rail Content</Segment>
+        </Rail>
+
+        <Rail dividing position='right'>
+          <Segment>Right Rail Content</Segment>
+        </Rail>
+
+        Segment content
```

Comment:
### SUI Docs

![image](https://cloud.githubusercontent.com/assets/5067638/16846616/f70311e4-49a1-11e6-9c85-309f361e4a0e.png)

### Stardust Docs

![image](https://cloud.githubusercontent.com/assets/5067638/16846639/1a79a818-49a2-11e6-9dde-f97f7b25b9b6.png)


Let's use the same paragraph image from the SUI docs to help illustrate the Rails.  Our docs will soon be hosted at `react.semantic-ui.com` anyway, so we'll be using lots of the standard images.

```js
<Image src='http://semantic-ui.com/images/wireframe/paragraph.png' />
```

This should do it ^ I think.